backend/scraper/scrape_walmart.py

```diff
@@ -11,12 +11,25 @@
 class WalmartScraper:
     zipSet = False
     def __init__(self):
-        chrome_options = Options()
-        chrome_options.add_argument('--headless')
-        try:
-            self.driver = webdriver.Firefox(executable_path=r'.\geckodriver.exe')
-        except:
-            sys.exit("Web Driver is missing")
+        if len(sys.argv) != 2:
+            raise ValueError('Please enter a browser driver to use (Chrome or Firefox)')
+
+        browser = sys.argv[1]
+        if browser.lower() == 'chrome':
+            chrome_options = Options()
+            chrome_options.add_argument('--headless')
+            try:
+                driver = r'./chromedriver.exe'

```

Comment:
Do you think you could use the WebDriver Manager, like Vicente did in #33? This PR has been open for a while, and using the manager isn't necessary for functionality, so we can do it in a separate PR, but eventually, it would be easier to use if the scripts manage their own drivers

Reply:
Yes, I can take a look and see if I can make it work.